Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+
+    
+    `LeapControllerOrientation`: Headset (Default) |  `LeapControllerOrientation`: Desk 
+    :-------------------------:|:-------------------------:
+    ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapHeadsetOrientationExampleMetacarpals.gif)  |  ![LeapDeskGif](../Images/CrossPlatform/LeapMotion/LeapDeskOrientationExampleMetacarpals.gif)
+    ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapDevManagerHeadset.png) |     ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapDevManagerDesk.png)
+
+1. Testing the Leap Motion Data Provider
+    - After Leap Motion Data Provider has been added to the input system profile, press play, move your hand in front of the Leap Motion Controller and you should see the joint representation of the hand.
+
+1. Building your project 
+    - Navigate to **File > Build Settings**
+    - Only Standalone builds are supported if using the Leap Motion Data Provider.
+
+
+## Removing Leap Motion from the Project
+
+1. Close Unity
+1. Close Visual Studio, if it's open
+1. Open File Explorer and navigate to the root of the MRTK unity project
+    - Delete the **UnityProjectName/Library** directory
+    - Delete the **UnityProjectName/Assets/LeapMotion** directory
+    - Delete the **UnityProjectName/Assets/LeapMotion.meta** file
+- Reopen Unity
+
+If errors are logged after reopening, restart unity again.
+
+Debugging
+
+1. Leap Namespace errors missing
+2. Obsolete errors 
+3. Editor errors
+
+
+## Getting the hand joints 
+
+Getting joints using the Leap Motion Data Provider is identical to hand joint retrieval for an MRTK Articulated Hand.  For more information, see [Hand Tracking](../Input/HandTracking.md).
+
+The following is a simple example of how to retrieve the pose of the palm joint.
+```
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+public class HandJointsLeap : MonoBehaviour
+{
+    private GameObject sphere;
+
+    private void Start()
+    {
+        sphere = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+        sphere.transform.localScale = Vector3.one * 0.03f;
+    }
+
+    private void Update()
+    {
+        // If the hand joint pose is available set the position of the sphere to the palm of the right hand
+        if (HandJointUtils.TryGetJointPose(TrackedHandJoint.Palm, Handedness.Right, out MixedRealityPose index))
+        {
+            sphere.transform.position = index.Position;
+        }
+    }
+}
+```
+
+## Leap Motion Example Scene
+
+The example scene uses the DefaultLeapMotionConfiguration profile and determines if the unity project has been configured correctly to use the Leap Motion Data Provider.
+
+LeapMotionHandTrackingExample scene location: MRTK/Examples/Demos/HandTracking/.  
+
+## See also
+
+* [Input Providers](../Input/InputProviders.md)
+* [Create a Data Provider](../Input/CreateDataProvider.md)
```

Comment:
I think that we might want to remove the links to 'create a data provider' and 'input simulation' because they aren't super pertinent to this - i.e. i read this as a consumer of leap, not as someone who wants to add a leap provider (this has already been done)